app/komponenter/samtykkepanel/SamtykkePanel.tsx

```diff
@@ -0,0 +1,57 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from '../tekstblokk/TekstBlokk';
+import css from './samtykkepanel.module.css';
+import { TypografiTyper } from '~/typer/typografi';
+
+interface Props {
+  tittel: SanityDokument;
+  innhold: SanityDokument;
+  samtykke: SanityDokument;
+  feilmelding: SanityDokument;
+  vedSamtykkeEndring: (bekreftet: boolean) => void;
+  feilmeldingAktivert: boolean;
+}
+
+const SamtykkePanel: React.FC<Props> = ({
+  tittel,
+  innhold,
+  samtykke,
+  feilmelding,
+  vedSamtykkeEndring,
+  feilmeldingAktivert,
+}: Props) => {
+  const [samtykkeErBekreftet, settSamtykkeErBekreftet] = useState(false);
+
+  const håndterSamtykkeEndring = (bekreftet: boolean) => {
+    settSamtykkeErBekreftet(bekreftet);
+    vedSamtykkeEndring(bekreftet);
+  };
+
+  return (
+    <div className={`${css.samtykkePanelOmråde}`}>
+      <TekstBlokk tekstblokk={tittel} />
+      <ConfirmationPanel
+        checked={samtykkeErBekreftet}
+        label={<TekstBlokk tekstblokk={samtykke} />}
+        onChange={() => {
+          håndterSamtykkeEndring(!samtykkeErBekreftet);
+        }}
+        error={
+          !samtykkeErBekreftet &&
+          feilmeldingAktivert && (
+            <TekstBlokk
+              tekstblokk={feilmelding}
+              typografi={TypografiTyper.BodyShort}
+            /> //denne gir feilmelding fordi den ikke er en ren String ("kan ikke være i <p>"). Visuelt fungerer den.
```

Comment:
🤯 Skal ikke denne ta inn React node? Er vi på siste versjon av `@navikt/ds-react`
<img width="234" alt="image" src="https://github.com/bekk/nav-familie-endringsmelding/assets/17828446/44e0ae43-ea24-4704-b8c3-5f04b743be0d">

Reply:
ConfirmationPanel skal ta inn en React Node som type i error, men for å hente tekst fra Sanity må vi bruke `<TekstBlokk>`